### PR TITLE
fix: Ensure all rules have a category tag

### PR DESCRIPTION
**Pull request by WilcoFiers**

### CODE REVIEW

**Comment by WilcoFiers** [comment]
Closes #772

File: doc/rule-descriptions.md [hunk 1]
```diff
@@ -23,24 +23,24 @@
...
-| landmark-one-main | Ensures a navigation point to the primary content of the page. If the page contains iframes, each iframe should contain either no main landmarks or just one | best-practice | true |
+| landmark-banner-is-top-level | The banner landmark should not be contained in another landmark | cat.semantics, best-practice | true |
+| landmark-contentinfo-is-top-level | The contentinfo landmark should not be contained in another landmark | cat.semantics, best-practice | true |
+| landmark-main-is-top-level | The main landmark should not be contained in another landmark | cat.semantics, best-practice | true |
+| landmark-no-duplicate-banner | Ensures the document has no more than one banner landmark | cat.semantics, best-practice | true |
+| landmark-no-duplicate-contentinfo | Ensures the document has no more than one contentinfo landmark | cat.semantics, best-practice | true |
+| landmark-one-main | Ensures a navigation point to the primary content of the page. If the page contains iframes, each iframe should contain either no main landmarks or just one | cat.semantics, best-practice | true |
```
Comment (dylanb) [inline doc/rule-descriptions.md]: Should this be `cat.keyboard`? Reason is that `bypass` is in that category and this is a subset of `bypass`

Comment (WilcoFiers) [reply doc/rule-descriptions.md]: I didn't think they all belonged in Keyboard, but I don't feel strongly about it either way. Let me know.

**Comment by WilcoFiers** [comment]
Talked to Dylan, we're leaving the tags as proposed.

**Pull request merged**